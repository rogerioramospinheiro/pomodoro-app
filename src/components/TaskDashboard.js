import React from 'react';
import TaskForm from './TaskForm';
import TimerDisplay from './TimerDisplay';
import FinishedTasks from './FinishedTasks';

const TaskDashboard = () => (
    <div>
        <TimerDisplay />
        <TaskForm />
        <FinishedTasks />
    </div>
);

export default TaskDashboard;