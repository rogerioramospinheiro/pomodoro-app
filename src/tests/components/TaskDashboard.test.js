import React from 'react';
import { shallow } from 'enzyme';
import TaskDashboard from '../../components/TaskDashboard';

test('Should render TaskDashboard', () => {
    // When
    const wrapper = shallow(<TaskDashboard />);
    // Then
    expect(wrapper).toMatchSnapshot();
})