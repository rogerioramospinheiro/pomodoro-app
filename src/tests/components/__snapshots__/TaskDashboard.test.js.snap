// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render TaskDashboard 1`] = `
<div>
  <Connect(TimerDisplay) />
  <Connect(TaskForm) />
  <Connect(FinishedTasks) />
</div>
`;
